draft-ietf-tls-esni.md

```diff
@@ -528,6 +530,58 @@ servers which have opted in, thus minimizing compatibility
 issues. However, there are two scenarios where that does not
 apply, as detailed below.
 
+## Load Balancing Multiple Independent Providers {#multi-cdn}
+
+The DNS is used to independently transfer two linked pieces of
+informaton to implement ESNI. The server is identified through DNS
+address records and the server's ESNI configuration is specified in a TXT
```

Comment:
does "address records" mean A or AAAA?

Reply:
yes.. I don't mind saying a/aaaa I was trying to leave room for other ways of specifying the address. srv? new A* type?